tvx/dummy_testingt.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+var t dummyTestingT
```

Comment:
this is duplicated both in top level and in chain/ ?

Reply:
Removed the top-level one - we now only have `require` and `assert` statements in `chain`.